src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -221,11 +228,10 @@ bool mongoConnect(const char* host, const char* db, const char* rplSet, const ch
 * mongoConnect -
 *
 * Version of the functions that doesn't uses authentication parameters
-*
 */
 bool mongoConnect(const char* host) {
 
-    return mongoConnect(host, "", "", "", "", false);
+    return mongoConnect(host, "", "", "", "", false, 0);
```

Comment:
I prefer not using defaults for the other function. The other function is the only one that will be used by the contextBroker binary and it should provide all the parameters. Note that this `mongoConnect(const char* host)` is only used in unit tests.